Assets/Scripts/Camera/FollowPlayer.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FollowPlayer : MonoBehaviour
+{
+    public GameObject PlayerTarget;
+    public Vector3 offset;
+    public float positionSmooth = 0.25f;
+
+    public float minZoom = 5.0f;
+    public float maxZoom = 10.0f;
+    public float maxVelocity = 15.0f;
+    public float zoomSmooth = 0.25f;
+
+    private Rigidbody2D player_rb;
+    private Camera cam;
+    private Vector3 currentVelocity;
+    private float currentSize;
+
+    void Start(){
+        player_rb = PlayerTarget.GetComponent<Rigidbody2D>();
+        cam = GetComponent<Camera>();
+    }
+    // Update is called once per frame
+    void LateUpdate()
+    {
+        transform.position = Vector3.SmoothDamp(
+                transform.position,
+                PlayerTarget.transform.position + offset,
+                ref currentVelocity,
+                positionSmooth
```

Comment:
Interpolacja jest zależna od framerate'u. Powinniśmy używać deltaTime. Prawdopodobnie powinno być `smooth^dt` albo `1-smooth^dt` (do sprawdzenia).

Reply:
Smoothdamp takes deltatime into account by default. It should already be framerate independent.

Source: https://docs.unity3d.com/ScriptReference/Vector3.SmoothDamp.html